namui/src/namui/common/types/macros.rs

```diff
@@ -142,6 +142,16 @@ macro_rules! common_for_f32_type {
                 $your_type(self as f32)
             }
         }
+
+        impl crate::SimpleSigned for $your_type {
```

Comment:
This should be $crate::SimpleSigned. Test it in other crate to make it sure.

Reply:
[4dabf20](https://github.com/NamseEnt/namseent/pull/379/commits/4dabf2076dec13b783e4e8c7059937e1af8cd1aa)